### ISSUE TITLE
Winit changes cause compilation failure

### ISSUE THREAD

**Issue by ZackGabri** [porglezomp/pixel-canvas#1]
So i have been trying to install pixel-canvas to give it a try, but when i try to build it, this error gets popped out into my terminal
```rs
error[E0599]: no method named `hidpi_factor` found for type `&winit::window::Window` in the current scope
   --> G:\Rust\.cargo\registry\src\github.com-1ecc6299db9ec823\pixel-canvas-0.2.0\src\canvas.rs:221:42
    |
221 |             display.gl_window().window().hidpi_factor()
    |                                          ^^^^^^^^^^^^ method not found in `&winit::window::Window`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0599`.
error: could not compile `pixel-canvas`.
warning: build failed, waiting for other jobs to finish...
error: build failed
```

i'm using the latest stable version of Rust with MSVC Build tools

**Comment by ZackGabri** [comment]
looked into `winit` docs for a few minutes and found out that `hidpi_factor()` is replaced by `scale_factor()`, so i got a local-copy for myself and modified that little mistake, hope you don't mind, and until you get this mistake fixed on the main crate, i will just serve myself out with the local-copy i got

**Comment by porglezomp** [comment]
Thanks for filing the issue! It should be fixed in v0.2.1 which I just published.

**Comment by ZackGabri** [comment]
Thanks!
